keel-titus-plugin/src/main/kotlin/com/netflix/spinnaker/keel/titus/batch/ContainerJobConfig.kt

```diff
@@ -39,23 +53,31 @@ data class ContainerJobConfig(
   val waitForCompletion: Boolean = true
 ) {
   init {
-    require((tag == null) xor (digest == null)) {
-      "One, and only one, of digest or tag must be supplied"
+    require((image == null) xor (repository == null)) {
+      "One, and only one, of image or repository must be supplied"
     }
+
     require(retries >= 0) {
       "Retries must be positive or zero"
     }
   }
 
-  /**
-   * ID that identifies an image, e.g.:
-   *
-   * acme/widget:latest
-   * acme/widget:sha256:780f11bfc03495da29f9e2d25bf55123330715fb494ac27f45c96f808fd2d4c5
-   */
-  val imageId: String = "$repository:${tag ?: digest}"
   val cloudProvider: String = "titus"
   val cloudProviderType: String = "aws"
+
+  /**
+   * Determine imageId depending on the newer field (image) or the deprecated fields (repository, tag)
+   */
+  val imageId : String
+  get() =
```

Comment:
Indentation is off here.

Reply:
I really wish spotless would catch this sort of thing. It passes spotlessCheck on my machine.